data-structures-queues/postpone.js

```diff
@@ -0,0 +1,8 @@
+/* exported postpone */
+
+function postpone(queue) {
+  if (queue.peek() === undefined);
+  else {
+    queue.enqueue(queue.dequeue());
+  }
```

Comment:
You never want to have empty code blocks, in this case your `if` code block is empty. So it would be better to write this as:
```JAVASCRIPT
  if (queue.peek() !== undefined) {
    queue.enqueue(queue.dequeue());
  }
```